edx_rbac/utils.py

```diff
@@ -114,8 +113,9 @@ def append_role_auth_claim(role_string, context=None):
             role_func = getattr(module, func_name)
         except (ImportError, AttributeError):
             # otherwise, assume that it's a django model
-            module = apps.get_model(module_name, func_name)
-            role_func = module.get_assignments
+            models = importlib.import_module('{0}.models'.format(module_name))
```

Comment:
whats a purpose of these changes plz add some explanation.

Reply:
description updated.

Reply:
You may not need this change after moving the new migrations to the `tests` directory.

Reply:
+1 to what Jeremy said.  Once the migration is moved into the `test_app/migrations` directory. I think you should be able to undo the changes in this file.